src/components/ProjectCard.js

```diff
@@ -1,133 +1,303 @@
 import React from 'react';
-import { makeStyles } from '@material-ui/styles';
-import Avatar from '@material-ui/core/Avatar';
+import { makeStyles, useTheme } from '@material-ui/styles';
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
-import Chip from '@material-ui/core/Chip';
 import Grid from '@material-ui/core/Grid';
-import FiberManualRecordRoundedIcon from '@material-ui/icons/FiberManualRecordRounded';
 import githubColorDictionary from './data/gh-colors.json';
+import { Typography } from "@material-ui/core";
```

Comment:
It's a minor issue, but in general when importing material ui components we try to use the following format and sort them alphabetically:

```import Typography from '@material-ui/core/Typography';```

Reply:
Change the import statement as suggested and sort them alphabetically in index.js(ProjectCard)